qml/KeyboardContainer.qml

```diff
@@ -64,7 +64,7 @@ Item {
         anchors.fill: parent
         asynchronous: true
         visible: panel.state === "EMOJI"
-        source: "languages/Keyboard_emoji.qml"
+        source: visible ? "languages/Keyboard_emoji.qml" : ""
```

Comment:
I would rather set the active property of the Loader to false

active: panel.state === "EMOJI"

Reply:
And then maybe visible: active (instead of repeating the code there).

source can then jsut stay like it is

Reply:
Changed.